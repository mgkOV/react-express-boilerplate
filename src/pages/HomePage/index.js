import React, { Component } from 'react';
import Helmet from 'react-helmet';

class HomePage extends Component {
  render() {
    return (
      <div>
        <Helmet>
          <title>My Title</title>
        </Helmet>
        <h1>HomePage</h1>
      </div>
    );
  }
}

export default HomePage;
