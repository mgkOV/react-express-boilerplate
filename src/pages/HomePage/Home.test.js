import React from 'react';
import { shallow } from 'enzyme';
import HomePage from './index';

describe('HomePage', () => {
  const homePage = <HomePage />;
  it('should render properly', () => {
    expect(homePage).toMatchSnapshot();
  });
});
