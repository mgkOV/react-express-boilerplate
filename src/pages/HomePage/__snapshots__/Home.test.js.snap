// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should render properly 1`] = `<HomePage />`;
